dusty/src/main.rs

```diff
@@ -39,15 +39,28 @@ struct Cmd {
     /// Print raw binary data along with summaries or headers
     #[structopt(short, long)]
     raw: bool,
+
+    /// Format output as JSON. Not compatible with `--raw`.
+    #[structopt(long)]
+    json: bool,
```

Comment:
Technically these are mutually exclusive flags. I looked through structopt's docs and I didn't see a way of indicating mutual exclusivity. I could convert this to clap and do it that way. Technically that'd be the _right_ way, especially since `structopt` is more or less deprecated for clap 3.

Reply:
Went ahead and did the clap upgrade